src/qmozcontext.cpp

```diff
@@ -99,6 +100,8 @@ class QMozContextPrivate : public EmbedLiteAppListener {
         }
 
         setDefaultPrefs();
+        // Set DPI used by APZC
+        mApp->SetDPI(QGuiApplication::primaryScreen()->logicalDotsPerInch());
```

Comment:
All platform dependent code in Gecko uses logical dpi, because theoretically user may set its own system-wide value for DPI. E.g. like here

https://github.com/nemomobile-packages/gecko-dev/blob/nemo/31.8.0.4/gfx/thebes/gfxQtPlatform.cpp#L226

By default logical dpi value is equal to physical one.